catalog/kafka/kafkacatalog.go

```diff
@@ -83,6 +83,10 @@ func GenDefaultCreateServiceRequest(platform string, region string, azs []string
 	// generate member ReplicaConfigs
 	replicaCfgs := genMemberConfigs(platform, cluster, service, azs, opts)
 
+	envkvs := []*common.EnvKeyValuePair{
```

Comment:
This would not work. Please follow the same way with zk, add -Dcom.sun.management.jmxremote.rmi.port=$JMX_PORT to KAFKA_JMX_OPTS in docker-entrypoint.sh

Reply:
defining envkvs here is not required. JMX_PORT is added into servicecfgfile. entrypoint.sh sources servicecfgfile already. Please see the comment to the entrypoint.sh